app/_landing_pages/ai-gateway.yaml

```
@@ -217,42 +226,91 @@ rows:
             cta:
               url: /advanced-analytics/
               align: end
+      - blocks:
+        - type: card
+          config:
+            title: Streaming
+            description: Streaming user requests with AI Gateway
```

```suggestion
            description: Stream user requests with AI Gateway
```
For consistency with the other descriptions